### ISSUE TITLE
pokefam stuff

### ISSUE THREAD

**Issue by taltamir** [soolar/sl_ascend#246]
At some point in the pokefam run it started doing

> Familiars can't be used in Pokefam.

After every turn for a few dozen turns before stopping. Seems like it was trying to use a familiar somewhere as a normal familiar? I didn't note where it was specifically and it seems this error was not saved to the logfile so I can't determine it. I will see if I can repeat it on additional runs and pin point where it happens. (so we can guess which familiar it is trying to use)

=====

During the defiled nook. every time it encountered the NC "skulls, skulls, skulls" it got stuck

> [194] The Defiled Nook
> Encounter: Skull, Skull, Skull
> [Insert Punch Out music here]
> Encounter: Skull, Skull, Skull
> Digimon hit a choice adventure (The Defiled Nook), trying....
> Encounter: Skull, Skull, Skull
> Digimon hit a choice adventure (The Defiled Nook), trying....
> Encounter: Skull, Skull, Skull
> Digimon hit a choice adventure (The Defiled Nook), trying....
> Encounter: Skull, Skull, Skull
> Digimon hit a choice adventure (The Defiled Nook), trying....
> Encounter: Skull, Skull, Skull
> Digimon hit a choice adventure (The Defiled Nook), trying....
> Encounter: Skull, Skull, Skull
> Digimon hit a choice adventure (The Defiled Nook), trying....
> Encounter: Skull, Skull, Skull
> Choice chain too long or I'm stuck!

**Comment by taltamir** [comment]
I think the lil doctor bag doesn't actually do anything in pokefam so it shouldn't be used. (the skills it gives cannot be used and the the HP bonus is useless).
Buffed stats also don't matter, should prioritize +xp items and then +item and +meat I think?

**Comment by soolar** [comment]
Not sure what's going on with the other stuff, but that's a fair point on the equipment priorities in pokefam. I'll be sure to keep them in mind while working on the new maximizer based equipment system.

If the other stuff hasn't mysteriously fixed itself by now and is making sl_ascend unusable for pokefam, I'll look in to it after I finish the maximizer based equipment system.

**Comment by taltamir** [comment]
Thanks, it is still usable even if a bit shaky. I had to take over manually a few times such as during the defiled nook. But once I finished that I was able to run it again.

Also to clarify further on the "familiar can't be used" error, when I said it stopped I meant the error. The error appeared during some quests for every turn, before stopping. I think its just that some specific quests are trying to equip a suitable familiar without checking if it is pokefam where familiars cannot be switched normally.
